lib/funky/video.rb

```diff
@@ -40,6 +40,11 @@ def picture
       data[:picture]
     end
 
+    # @return [String] the name of Facebook page for the video.
+    def page_name
+      data.dig(:from, :name)
```

Comment:
This is 🆗 

However this method has been added to recent versions of Ruby, so if we go with this code, we should add the required Ruby version both to the README and to the `.gemspec` file.

@philipqnguyen Are you okay with not supporting older versions of Ruby?